adapters/between/usersync_test.go

```diff
@@ -0,0 +1,8 @@
+package between
+
+import (
+	"testing"
+)
+
+func TestBetweenSyncer(t *testing.T) {
+}
```

Comment:
From the rest of the code it looks like your adapter doesn't support usersyncing and so can you please remove this file and the `usersync.go` file? Thanks